packages/doenetml-worker-rust/lib-doenetml-core/src/core/general_prop/independent_prop.test.rs

```diff
@@ -125,16 +125,16 @@ mod setup_functions {
 
         // Note: the default_value specified in the creation of prop above
         // isn't used at this level of testing, so we supply it directly here to match
-        let (preliminary_value_dependency, preliminary_value_var) =
+        let (independent_state_dependency, independent_state_var) =
```

Comment:
`state_var` -> `prop`

Reply:
Trying to distinguish between the "independent state prop" that is the leaf of the dependency graph and the "independent prop" that is the actual prop that mirrors the state.  I'm trying "independent_state" for the former.